ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committer/ConfigCommitter.scala

```diff
@@ -0,0 +1,181 @@
+// Copyright (c) 2019 The DAML Authors. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.participant.state.kvutils.committer
+
+import com.daml.ledger.participant.state.kvutils.Conversions._
+import com.daml.ledger.participant.state.kvutils.DamlKvutils._
+import com.daml.ledger.participant.state.kvutils.committing.Common.getCurrentConfiguration
+import com.daml.ledger.participant.state.v1.Configuration
+
+private[kvutils] object ConfigCommitter {
+  case class Result(
+      builder: DamlConfigurationEntry.Builder,
```

Comment:
make this `DamlConfigurationSubmission` as we're not really using the builder to gradually build up the result.